src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-flow: row wrap;
+  font-family: $font-family;
+  font-size: $font-size;
+  gap: $grid-gap;
+  padding-inline: $calendar-inline-padding;
+  width: $calendar-width;
+
+  &__day {
+    align-items: center;
+    background-color: $day-color;
+    border: $day-border;
+    box-sizing: border-box;
+    cursor: pointer;
+    display: flex;
+    height: $day-size;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)